Tests/Clappr_Tests/Classes/Plugin/Core/MediaControl/MediaControlPluginTests.swift

```diff
@@ -0,0 +1,61 @@
+import Quick
```

Comment:
This file was not added to the project.

Reply:
The file was renamed.

Reply:
Ok. 👍 So `MediaControlPluginTests.swift` should be deleted.